js/controller.js

```diff
@@ -604,13 +612,17 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
     },
 
-    onFullscreenChanged: function(event, fullscreen, paused) {
+    onFullscreenChanged: function() {
+      if (this.state.isFullScreenSupported) {
```

Comment:
This IF block is checked again inside toggeFullScreen method: if (this.state.isFullScreenSupported)
We can refactor this.state.fullscreen = Fullscreen.isFullscreen; into toggleFullScreen()